docs/commands/cd.md

```diff
@@ -0,0 +1,25 @@
+# cd
+
+If you don't know about it already, the `cd` command is very simple. It stands for 'change directory' and it does exactly that. It changes the current directory that you are in to the one specified. If no directory is specified, it takes you to the home directory. Additionally, `..` takes you to the parent directory
+
+## Examples -
+
+```shell
+/home/username> cd Desktop
+/home/username/Desktop> now your current directory has been changed
+```
+
+```shell
+/home/username/Desktop/nested/folders> cd ..
+/home/username/Desktop/nested> cd ..
+/home/username/Desktop> cd ../Documents/school_related
+/home/username/Documents/school_related> cd ../../..
+/home/>
+```
+
+```shell
```

Comment:
Looks good!

For examples 3 and 4 you could give an example of using `cd ~` to go home and `cd -` to go to the previous directory.

Reply:
Hmm... neither of those commands work on my Manjaro Linux laptop
![Screenshot from 2019-10-02 06-21-00](https://user-images.githubusercontent.com/32027323/66010625-62eb3d80-e4dd-11e9-9ff2-4e9004a5b0c3.png)

`nu --version` gives : 

```
nushell 0.1.3
```

Reply:
@coolshaurya hi! How did you install `nu`? 

Can you run `nu` and use the `version` command? what does it say?

Reply:
i installed nu using the `nushell-bin` AUR package

version in the nu shell gives -

```
---------
 version 
---------
 0.2.0 
---------
```

Reply:
@coolshaurya Yeah, `cd ~` and `cd -` were landed after that release. This is why you are seeing errors instead.